<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rania</class>
 <widget class="QMainWindow" name="rania">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="fond">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1211</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_8"/>
    <widget class="QWidget" name="page_6">
     <widget class="QStackedWidget" name="mainwidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>1141</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_nom">
        <property name="geometry">
         <rect>
          <x>134</x>
          <y>55</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom">
        <property name="geometry">
         <rect>
          <x>114</x>
          <y>105</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Prenom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_CIN">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>170</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nbconsultation">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Nombre De Consultation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nbnuit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Nombre De Nuits :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_nom">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_prenom">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_CIN">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_consultation">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>220</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_nuit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>270</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmer_ajout">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_convention">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>340</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_conventionajout">
        <property name="geometry">
         <rect>
          <x>64</x>
          <y>340</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Convention</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QTableView" name="tableView_affichage">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>981</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>21</y>
          <width>361</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_sort">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>27</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort Consultation</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_supp">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>211</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN Ã  supprimer :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmer_supp">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QRadioButton" name="radioButton_nom">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_prenom">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>440</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_modifier">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_newval">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>170</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>New Value</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_cin">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_consultation">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>50</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>NB Consultation</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_nuit">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>50</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nb Nuite</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_15">
       <widget class="QLineEdit" name="subject">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>341</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Objet...</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="msg">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>611</width>
          <height>201</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Message...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Envoyer_">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>350</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Email_">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>341</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Email...</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="ajout">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QPushButton" name="pushButton_ajoutConsultation">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajout</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_Consultation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>1151</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_9"/>
      <widget class="QWidget" name="page_10">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_NomSociete">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom Societe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Remise">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>190</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NomSociete">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Remise">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_11">
       <widget class="QTableView" name="tableView_Consultation">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>1121</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_13">
       <widget class="QPushButton" name="pushButton_Supp">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_suppConsultation">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_consultationsupp">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Id :</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_14">
       <widget class="QRadioButton" name="radioButton_nomSociete">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom </string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_remise">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>40</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ModifierConsultation_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idconsultation">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_valConsultation">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>200</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>new val</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_afficherConsultation">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SupprimerConsultation">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ModifierConsultation">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_12"/>
   </widget>
   <widget class="QPushButton" name="pushButton_convention">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Convention</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reservation">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Reservation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
